common/src/main/java/com/microsoft/identity/common/internal/dto/AccessTokenRecord.java

```diff
@@ -157,6 +169,24 @@ public void setKid(@Nullable final String kid) {
         mKid = kid;
     }
 
+    /**
+     * Gets the requested_claims hash.
+     * @return The hashed copy of the claims string.
+     */
+    public String getRequestedClaims() {
+        return mRequestedClaims;
+    }
+
+    /**
+     * Hashes the incoming string (we don't know the length or potential special characters)
```

Comment:
I am unclear about the contents of this javadoc -- this being setter, I wouldn't expect any hashing to occur here, nor for it to me mentioned unless this value is expected to be hash prior to being set (in which case this is a statement about the input data)

Reply:
Thanks for pointing this out. I wrote the comment in an early iteration when I was hashing in the setter (I since realized the error in my ways!) 

Fixed in latest commit